vyper/venom/passes/sccp/sccp.py

```diff
@@ -329,6 +338,40 @@ def _replace_constants(self, inst: IRInstruction):
                 if isinstance(lat, IRLiteral):
                     inst.operands[i] = lat
 
+    def _fix_phi_nodes(self):
+        visited: OrderedSet[IRBasicBlock] = OrderedSet()
```

Comment:
why keep track of visited, instead of just iterating over `fn.get_basic_blocks()`?